plugins/pip.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "pip",
+    "version": "0.0.1",
+    "match": "^python3[0-9]{1,2}Packages.pip",
+    "readme": "This plugin adds a script for automatically creating a virtual environment using `venv` for python3 projects, so you can install packages with pip as normal.\nTo activate the environment, run `source $VENV_DIR/bin/activate` or add it to the init_hook of your devbox.json\nTo change where your virtual environment is created, modify the $VENV_DIR environment variable in your init_hook",
+    "env": {
+        "VENV_DIR": "{{ .Virtenv }}/pip/.venv"
```

Comment:
`pip` is not needed `Virtenv` is already specific to this package.  This can be `"VENV_DIR": "{{ .Virtenv }}/.venv"`